pkg/cgroups/cgroups.go

```diff
@@ -10,14 +10,17 @@ import (
 	"strconv"
 	"strings"
 
+	"github.com/containers/libpod/pkg/rootless"
 	spec "github.com/opencontainers/runtime-spec/specs-go"
 	"github.com/pkg/errors"
 	"github.com/sirupsen/logrus"
 )
 
 var (
 	// ErrCgroupDeleted means the cgroup was deleted
-	ErrCgroupDeleted = errors.New("cgroups: cgroup deleted")
+	ErrCgroupDeleted = errors.New("cgroup deleted")
+	// ErrCgroupV1Rootless means the cgroup v1 were attempted to be used in rootless environmen
+	ErrCgroupV1Rootless = errors.New("CGroups V1 not supported in rootless environment")
```

Comment:
can we change this to "the CGroups V1 API..."? Makes the error message start with a lowercase letter which makes Go happier